docs/troubleshooting.md

```diff
@@ -0,0 +1,83 @@
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+# Troubleshooting
+
+## Where is the Uniffle log file?
+
+Uniffle logs are stored in the `$RSS_LOG_DIR`, which defaults to `${RSS_HOME}/logs`. The common log file names are `coordinator.log`, `shuffle-server.log`, `dashboard.log`.
+
+## Audit logs
+
+The Uniffle cluster provides audit logs for each process. You can also find audit logs in the log directory, the log file names are `coordinator_rpc_audit.log`, `shuffle_server_rpc_audit.log`, `shuffle_server_storage_audit.log`.
+
+| Audit log name                   | Configuration                         | Default | Description                                                                 |
+|----------------------------------|---------------------------------------|---------|-----------------------------------------------------------------------------|
+| coordinator rpc audit log        | rss.coordinator.rpc.audit.log.enabled | true    | Record coordinator rpc operation audit.                                     |
+| shuffle server rpc audit log     | rss.server.rpc.audit.log.enabled      | true    | Record shuffle server rpc operation audit.                                  |
+| shuffle server storage audit log | rss.server.storage.audit.log.enabled  | false   | The server will log audit records for every disk write and delete operation |
+
+Based on the above audit logs, you can check the operation details and the operation time cost.
+
+## Uniffle remote debug
+
+### Debugging Uniffle processes
+
+Java remote debugging makes it easier to debug Uniffle at the source level without modifying any code. You will need to set the JVM remote debugging parameters before starting the process. There are several ways to add the remote debugging parameters; you can export the following configuration properties in shell or conf/rss-env.sh:
+
+```shell
+# Java 8
+export DASHBOARD_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5004"
+export COORDINATOR_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5006"
+export SHUFFLE_SERVER_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005"
+
+# Java 11
+export DASHBOARD_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=*:5004"
+export COORDINATOR_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=*:5006"
+export SHUFFLE_SERVER_JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=*:5005"
+```
+
+In general, you can use `<PROCESS>_JAVA_OPTS` to specify how an Uniffle process should be attached to.
+
+suspend={y | n} will decide whether the JVM process waits until the debugger connects or not.
+
+address determines which port the Uniffle process will use to be attached to by a debugger. If left blank, it will choose an open port by itself.
```

Comment:
```
The `address` parameter determines which port the Uniffle process will use to be attached to by a debugger. If left blank, it will choose an open port by itself.
```